src/Documentation/sidebar.json

```diff
@@ -47,15 +47,13 @@
         "label": "Share Data & Model Files",
         "slug": "share-data-and-model-files"
       },
-      {
-        "label": "Shared Development Machine",
-        "slug": "multiple-data-scientists-on-a-single-machine"
-      }
+      "update-tracked-files",
```

Comment:
Not sure whether [Update Tracked Files](https://dvc-org-pr-637.herokuapp.com/doc/use-cases/update-tracked-files) as is belongs to Use Cases or User Guide. I think it's more of a use case. @shcheklein thinks it's more of a User Guide. Informal definitions:

> Use Cases: (the most blog post-like like content, lots of text about motivation, titles that match real-life workflow/high level cases, has links to 1+ user guide sections and 1+ links to commands reference)

> User Guide: like a car owner manual: not a goal to explain even why do you need a car, but explain different parts, how to use them. Of course it can include some motivation (like why would you want to manage external data?). But in general it’s more technical, dry.

Reply:
p.s. note that I've updated the index.md of each doc section to include a formal description of what "use cases" and "user guides" are. (See [UC](https://dvc-org-pr-637.herokuapp.com/doc/use-cases) and [UG](https://dvc-org-pr-637.herokuapp.com/doc/user-guide), respectively.)

Reply:
just to clarify, do we need to clarify further the difference between UC/UG? is it clear enough now why "updating a tracked file" is more about UG and not UC? if not, could you please give a bit more context why do you consider it as a UC?

Reply:
re User Guide vs. Use Cases:

I see why you don't want **Update Tracked Files** in Use Cases. It's not specific enough real-file example. But TBH the other Use Cases are also general in some ways, especially [Share Data and Model Files](https://dvc.org/doc/use-cases/share-data-and-model-files), which example S3 could be replaced by any other supported remote type. The existing use cases just happen to include a real-life example and some diagrams. Yet, for consistency (since UTF has no example or diagrams) I agree with you... So I'll move it back for now.

BUT I also don't think it really belongs in the User Guide, which "describes the main DVC concepts and features comprehensively... These guides don't focus on specific scenarios, but have a general scope".

Anyway. I'm opening a new issue to solve this confusion once and for all! See #654

Reply:
Agreed regrading S3, it was used for illustration only and probably this document should mention that there are other remotes.

Also, agreed that sometimes it's hard to make it precisely "Use Casey".

Regarding the User Guide - there are sections like "Large Files Optimization" - it's not a use case by itself, it's a technical description how to fine fine tune DVC in certain situations. I think UG should include things like `Pipelines -> How to Edit, How to Create`, etc. And Updating Tracked Files is very similar in my mind to those.

Reply:
> Regarding the User Guide...

Good points. Moved discussion to https://github.com/iterative/dvc.org/issues/654#issuecomment-536330090 🙂

Reply:
> ...and probably this document should mention that there are other remotes.

 Added note in e29233b